packages/react-components/src/components/chart/tests/useResize.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { describe, expect } from '@jest/globals';
+import { mockSeries, mockSize, mockViewport } from './getTrendCursor.spec';
+import { renderHook } from '@testing-library/react';
+import useResize from '../hooks/useResize';
+import { InternalGraphicComponentGroupOption } from '../types';
+describe('useResize', () => {
```

Comment:
Nit: Spacing.

Reply:
not positive on this, it looks... fine in local